### PR TITLE
Added functional DeAnza spider

### PR DESCRIPTION
**Pull request by HarrisonGregg**

### CODE REVIEW

File: osp_scraper/spiders/deanza.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+from ..spiders.CustomSpider import CustomSpider
+
+class DeAnzaSpider(CustomSpider):
+    name = "deanza"
+
+    def start_requests(self):
+        start_url = "http://ecms.deanza.edu/deptoutlinespublic.html"
+
+        def get_departments(response):
+            for option in response.css("#Dept option"):
```
Comment (jodizzle) [inline osp_scraper/spiders/deanza.py]: The first option in the department list looks like it has an empty value, so you should skip it, e.g.,
```python
for option in response.css("#Dept option")[1:]:
```

Comment (HarrisonGregg) [inline osp_scraper/spiders/deanza.py]: Good catch.

File: osp_scraper/spiders/deanza.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                code = option.css("option::attr(value)").extract_first()
+                if code:
+                    name = option.css("option::text").extract_first()
```
Comment (jodizzle) [inline osp_scraper/spiders/deanza.py]: Is this variable ever used?

**Comment by jodizzle** [comment]
OK to merge (though you might want to clean up trailing whitespace and the like).

**Pull request merged**